packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -564,6 +635,8 @@ class TextStyle extends Diagnosticable {
   @override
   void debugFillProperties(DiagnosticPropertiesBuilder properties, { String prefix: '' }) {
     super.debugFillProperties(properties);
+    if (debugLabel != _kDefaultDebugLabel)
+      properties.add(new StringProperty('${prefix}debugLabel', debugLabel, defaultValue: null));
```

Comment:
probably `showName: false`? Or maybe this is a MessageProperty? Check to see what @jacob314 did with the other debugLabels.

Reply:
showName: false sounds reasonable. I don't see any other debugLabel properties added as diagnosticProperty. Generally, if you thing debugLabel does conceptually represent a property name, StringProperty is what you want. If debugLabel is just a debugging message you want to show than MessageProperty is what you want.

Reply:
Thanks! Switched to `MessageProperty`.